### DISCUSSION TITLE
关于DesignAble会不会有一份API文档呢?

### DISCUSSION THREAD

**Discussion by sorryljt** [alibaba/formily#1927] [Q&A 2.x] [answered]
#### 如题

现在在基于Formily和DesignAble开发，有些扩展要去基于其DesignAble源码去探索或者猜测用法，用起来比较吃力，希望有一份基于DesignAble的文档

**Comment by janryWang** [comment, marked as answer]
会有的，需要时间

**Reply by sorryljt** [reply]
那麻烦大佬看一下下一个关于jsonSchema反显在DesignAble 设计页面上的问题，目前还没有跑通demo

**Reply by janryWang** [reply]
> 那麻烦大佬看一下下一个关于jsonSchema反显在DesignAble 设计页面上的问题，目前还没有跑通demo

现在先自己研究吧，目前没时间答疑designable使用相关的问题
